src/backend/looped/self/super_hero_fly.cpp

```diff
@@ -289,7 +297,7 @@ namespace big
 			}
 
 			if (PAD::IS_DISABLED_CONTROL_PRESSED(0, (int)ControllerInputs::INPUT_COVER) && !m_flying
-			    && g.self.super_hero_fly.charge)
+			    && g.self.super_hero_fly.charge && !PED::IS_PED_IN_ANY_VEHICLE(self::ped, false))
```

Comment:
This change doesn't make any sense, you added a return check at the head of the function to see if you're in any vehicle, why are you checking it again?

Reply:
Funny thing, it still won't work lol